oak-solr-osgi/pom.xml

```diff
@@ -178,6 +178,12 @@
             <artifactId>commons-math3</artifactId>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.yetus</groupId>
+            <artifactId>audience-annotations</artifactId>
+            <version>0.5.0</version>
+            <scope>runtime</scope>
+        </dependency>
```

Comment:
Why is this additional dependency needed?

Reply:
this has been added in zookeeper 3.4.14. If we don't declare it as runtime dependency, osgi-it tests will fail.

Reply:
I was able to declare it as optional. Dependency removed.